src/ImageSharp/Formats/Jpg/Components/Block8x8F.cs

```diff
@@ -15,6 +15,11 @@ namespace ImageSharp.Formats
     /// </summary>
     internal partial struct Block8x8F
     {
+        // Most of the static methods of this struct are instance methods by actual semantics: they use Block8x8F* as their first parameter.
```

Comment:
Why not make them instance methods? Sounds a lot nicer, since that is the intention. Static is also generally harder to test.